### DISCUSSION TITLE
Action not being run

### DISCUSSION THREAD

**Discussion by kaihendry** [community/community#27219] [Actions]
<p>Hi, I’ve committed a couple of fixes to my workflow file:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/kaihendry/aws-partners-datasette/commits/main" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/765871/181102275-33916d99-edbc-4c45-8723-8a5ac67eaf8d.jpeg" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/kaihendry/aws-partners-datasette/commits/main" target="_blank" rel="noopener nofollow ugc">kaihendry/aws-partners-datasette</a></h3>

<p>My first datasette 🤣. Contribute to kaihendry/aws-partners-datasette development by creating an account on GitHub.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>And the last couple of commits appear not to have triggered a build:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/kaihendry/aws-partners-datasette/actions" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/765871/181102276-ae580004-90a9-416b-829a-1c42bb59b647.jpeg" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/kaihendry/aws-partners-datasette/actions" target="_blank" rel="noopener nofollow ugc">kaihendry/aws-partners-datasette</a></h3>

<p>My first datasette 🤣. Contribute to kaihendry/aws-partners-datasette development by creating an account on GitHub.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Why?</p>
<p>Thank you in advance,</p>

**Comment by kingthorin** [comment]
<p>The issue is displayed here: <a href="https://github.com/kaihendry/aws-partners-datasette/actions/runs/400116526" rel="noopener nofollow ugc">https://github.com/kaihendry/aws-partners-datasette/actions/runs/400116526</a></p>
<p>It seems at L36 you have too much of an indent.</p>

**Comment by kaihendry** [comment]
<p>There has been two commits from <code>af02a81</code> where it moaned about L36. So why isn’t it taking the newer commits into the pipeline?</p>
<p>I really wish pushed commits were just rejected if the YAML like this is invalid…</p>

**Comment by kaihendry** [comment, marked as answer]
<p>Fix was a painful waste of my life time <a href="https://github.com/kaihendry/aws-partners-datasette/commit/b13eeddd82a98f06af56ca221122d261d8551803" rel="noopener nofollow ugc">https://github.com/kaihendry/aws-partners-datasette/commit/b13eeddd82a98f06af56ca221122d261d8551803</a></p>
<p>I hope whoever renamed master to main is happy.</p>
